eRecruiter.Api.Client/Requests/Applicant/ApplicantGlobalIdGetRequest.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
+using System.Net.Http;
+using eRecruiter.Api.Responses;
+
+namespace eRecruiter.Api.Client.Requests
+{
+    public class ApplicantGlobalIdGetRequest : HttpRequestMessage<IEnumerable<ApplicantGlobalIdResponse>>
+    {
+        public ApplicantGlobalIdGetRequest(int globalId)
+            : base(HttpMethod.Get, "Api/Applicant/GlobalId/" + globalId)
```

Comment:
We should handle this like the e-mail address and use ``Api/Applicants/ByGlobalId/{email}`` to query for the golbal ID in one mandator.